frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -539,15 +585,22 @@ export default function InteractiveWaveformModal({
       // only filter if well data has been fetched, otherwise assume no filtering required because user would not have been able to have moved min peak and max valley lines
       if (well in originalData.coordinates) {
         const wellCoords = originalData.coordinates[well];
-        wellPeaks = wellPeaks.filter((peak) => wellCoords[peak][1] >= peakValleyWindows[well].minPeaks);
-        wellValleys = wellValleys.filter(
-          (valley) => wellCoords[valley][1] <= peakValleyWindows[well].maxValleys
-        );
+        wellPeaks = wellPeaks.filter((peak) => {
+          const actualY = wellCoords[peak][1];
+          const computedY =
```

Comment:
I think actualY and computedY are confusing names. I'm assuming `actualY` is referring to the y-val of the peak/valley and `computedY` is referring the y-val of the filter line at that same x-val